include/arch/riscv/arch/machine.h

```diff
@@ -16,6 +16,24 @@
 #include <arch/sbi.h>
 #include <mode/machine.h>
 
+/* PPTR_BASE must be smaller than KERNEL_ELFBASE. */
+compile_assert(pptr_base_less_elf_base, PPTR_BASE < KERNEL_ELF_BASE_RAW);
+/* physBase must be aligned to a page for verification to succeed. */
+compile_assert(phys_base_page_aligned, IS_ALIGNED(PHYS_BASE_RAW, seL4_PageBits));
```

Comment:
The specific condition is because the init state example in verification uses a page for the kernel ELF window, so in that sense a verification artifact. But I don't think it would make much sense to try to define a virtual address window that is not at least page aligned in any case, so IMO this condition should be checked regardless. (You could theoretically set the constant to a non-aligned value, but you're still going to have to map at least a page)